jack-core/src/com/rapleaf/jack/queries/AggregatedColumn.java

```diff
@@ -7,10 +7,21 @@
   }
 
   private final Function function;
+  private final String sqlKeyword;
+  private final String alias;
 
   private AggregatedColumn(Column column, Function function) {
-    super(column);
+    super(column.table, column.field, column.type);
```

Comment:
@roshan, I deleted the copy constructor because there was only one usage of that constructor. I am kind of opinionated towards keeping as few overloaded methods as possible. In the future, if there is more use case, we can add it back. Does it sound good to you?

Reply:
Sorry, I just mean that there's [this constructor](https://github.com/LiveRamp/jack/pull/236/files#diff-7fd887cbe87236ce962da94adf3a6578R20) which can be used here as:

```
this(
  column,
  function,
  function.toString() + "(" + column.getSqlKeyword() + ")",
  this.alias = column.getSqlKeyword().replaceAll("\\.", "_") + "_" + function.name().toLowerCase()
)
```

Right?

Reply:
Oh, right. Will fix.